rules/windows/registry_event/sysmon_comhijack_sdclt.yml

```diff
@@ -0,0 +1,26 @@
+title: COM Hijack via Sdclt
+id: 07743f65-7ec9-404a-a519-913db7118a8d
```

Comment:
This rule already pushed in repo (https://github.com/Neo23x0/sigma/blob/master/rules/windows/registry_event/sysmon_comhijack_sdclt.yml). Could you delete this rule from the PR?